docs/getting-started/quick-start-guide.md

```diff
@@ -24,8 +24,133 @@ To follow along with this guide you'll need to have these things installed:
+
+Launch the create platformatic command:
+
+```bash
+npm create platformatic@latest 
+```
+
+This starts the Platformatic creator wizard, which asks you some questions on how to create the Platformatic project. 
+For this quick-start, you should answer the questions as follows:
+
+- Which kind of project do you want to create?  => DB
+- Where would you like to create your project?  => quick-start 
+- Do you want to create default migrations?     => Yes
+- Do you want to create a plugin?               => Yes
+- Do you want to use TypeScript?                => No
+- Do you want to install dependencies?          => Yes (this can take a while)
+- Do you want to apply the migrations?          => Yes
+- Do you want to create the github action to deploy this application to Platformatic Cloud? => No
+
+:::info
+
+Please feel free to answer the questions differently, if you want to create a different kind of project.
+Just make sure to run the `npm install` manually if not done through the wizard.
+
+:::
+
+Now you have a Platformatic project in the folder `quick-start` with a default migration and a plugin.
+
+## Check the  database schema
+
+In your project directory (`quick-start`), open the `migrations` directory that can store your database migration files that will contain both the `001.do.sql` and `001.undo.sql` files. The `001.do.sql` file contains the SQL statements to create the database objects, while the `001.undo.sql` file contains the SQL statements to drop them.
+
+```sql title="migrations/001.do.sql"
+CREATE TABLE IF NOT EXISTS movies (
+  id INTEGER PRIMARY KEY,
+  title TEXT NOT NULL
+);
+```
+
+Note that this migration has been already applied by Platformatic creator.
+
+## Check your API configuration
+
+In your project directory, check the Platformatic configuration file named
+**`platformatic.db.json`**:
+
+```json title="platformatic.db.json"
+{
+  "server": {
+    "hostname": "{PLT_SERVER_HOSTNAME}",
+    "port": "{PORT}",
+    "logger": {
+      "level": "{PLT_SERVER_LOGGER_LEVEL}"
+    }
+  },
+  "core": {
+    "connectionString": "{DATABASE_URL}",
+    "graphql": true,
+    "openapi": true
+  },
+  "migrations": {
+    "dir": "migrations"
+  },
+  "plugin": {
+    "path": "plugin.js"
+  },
+  "types": {
+    "autogenerate": true
+  }
+}
+```
+
+...and the environment file named **`.env`**:
+
+```bash title=".env"
+PLT_SERVER_HOSTNAME=127.0.0.1
+PORT=3042
+PLT_SERVER_LOGGER_LEVEL=info
+DATABASE_URL=sqlite://./db.sqlite
+```
+
+This configuration tells Platformatic to:
+
+- Run an API server on `http://127.0.0.1:3042/`
+- Connect to an SQLite database stored in a file named `db.sqlite`
+- Look for database migration files in the `migrations` directory
+- Load the plugin file named `plugin.js` and automatically generate types
+
+:::tip
+
+The [Configuration reference](/docs/reference/db/configuration.md) explains all of the
+supported configuration options.
+
+:::
+
+## Start your API server
```

Comment:
Let's take users straight from the CLI tool to spinning up their API right away — they can learn about how it works after. I think it would be good if this section was before the two sections above.